### ISSUE TITLE
[Personal Account] Cancel confirmation pop-up is not  accessible for keyboard navigation (Tab + Enter)

### ISSUE THREAD

**Issue by Olgatariel** [ita-social-projects/GreenCity#8595]
**Environment:** macOS 14.6.1; Safari 17.6
**Reproducible:** always
**Build found:** last commit from https://github.com/ita-social-projects/GreenCityClient/tree/dev

**Preconditions**
1.The user is logged in
2. The user has an unpaid order with status “Сформовано” and payment status “Не оплачено”
3. The user is on the “Особистий акаунт" page

**Steps to reproduce**
1. Go to the 'Поточні замовлення' tab
2. Click on the “Скасувати” (Cancel) button for any order
3. Try to navigate between “Ні” and “Так” buttons using 'Tab' and 'Shift + Tab'      
4. Try to activate a selected button (e.g. “Ні”) using the 'Enter' key

**Actual result**
Tab does not normally focus the buttons. Enter key does not trigger any action. Full keyboard control of the pop-up is not working

**Expected result**
-User can switch between “Ні” and “Так” using Tab / Shift + Tab
-When the focus is on a button, pressing Enter should trigger the corresponding action

**User story and test case links**
User story #3488
Test case #8481 

**Labels to be added**
"Bug", Priority ("pri: "), Severity ("severity:"), Type ("UI, "Functional"), "API" (for back-end bugs).

**Comment by witolDark** [comment]
Cannot reproduce. Buttons are available using tab but not in first order. Yet, i can change it

**Comment by bohdana13** [comment]
fixed, build - 11.10.25
